registry/storage/driver/hdfs/driver.go

```diff
@@ -0,0 +1,449 @@
+package hdfs
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"fmt"
+	"github.com/colinmarc/hdfs"
```

Comment:
This import should be in the group below

Also looks like `go.mod` needs to be updated for the new dependency (and vendored)

Reply:
OK